proposals/sandbox/k3s.md

```diff
@@ -0,0 +1,471 @@
+| github.com/coreos/go-iptables/LICENSE | Apache-2.0 |
+| github.com/coreos/go-iptables/NOTICE | Confidence threshold not high enough for any known license |
+| github.com/coreos/pkg/LICENSE | Apache-2.0 |
+| github.com/coreos/pkg/NOTICE | Confidence threshold not high enough for any known license |
+| github.com/inconshreveable/mousetrap/LICENSE 1.0 | Apache-2.0 |
+| github.com/rubiojr/go-vhd/LICENSE | MIT |
+| github.com/gogo/googleapis/LICENSE | Apache-2.0 |
+| github.com/gogo/protobuf/LICENSE | BSD-3-Clause |
+| github.com/mattn/go-sqlite3/LICENSE | MIT |
+| github.com/mattn/go-shellwords/LICENSE | MIT |
+| github.com/chai2010/gettext-go/LICENSE  | BSD-3-Clause |
+| github.com/rootless-containers/rootlesskit/LICENSE | Apache-2.0 |
+| github.com/pkg/errors/LICENSE  | BSD-2-Clause |
+| github.com/munnerz/goautoneg/LICENSE 0.953917 | BSD-3-Clause |
+| github.com/ghodss/yaml/LICENSE | Confidence threshold not high enough for any known license |
+| github.com/vmware/govmomi/vim25/xml/LICENSE  | BSD-3-Clause |
+| github.com/vmware/govmomi/LICENSE.txt| Apache-2.0 |
+
+Additionally, k3s ships with or leverages the following technologies:
+| Technology | License |
+| ------------- | ------------- |
+| https://github.com/containous/traefik | MIT |
+| https://github.com/helm/helm | Apache 2.0 |
+| https://github.com/coredns/coredns | Apache 2.0 |
+| https://github.com/buildroot/buildroot | GPL v2 | 
+| https://github.com/mirror/busybox | GPL v2
+
+_What actions are required to be compliant with the IP policy?_
+
+This is under review.
+
+## Other Considerations
+### _Please note, these are not gating criteria but rather to:_
+- Collect a standard set of information for each project
+- Provides a point in time capture of the state of the project which makes it easier to track progress at future reviews and / or promotion
+- Help projects to prepare for SIG and TOC presentation
+- Allow the SIG to review the project and perform due diligence for incubation
+- Provide the TOC with the information required to accept sponsorship of a project and/or votes
+- Identify and rectify any significant issues / blockers prior to presenting to the TOC and acceptance as a CNCF project
+
+### Cloud Native
+_Does the project meet the definition of Cloud Native? The CNCF charter states:_
+```
+“Cloud native technologies empower organizations to build and run scalable applications in modern, dynamic environments such as public, private, and hybrid clouds. Containers, service meshes, microservices, immutable infrastructure, and declarative APIs exemplify this approach.
+
+
+“These techniques enable loosely coupled systems that are resilient, manageable, and observable. Combined with robust automation, they allow engineers to make high-impact changes frequently and predictably with minimal toil.”
+```
+
+Yes.
+
+### Project and Code Quality
+_Are there any metrics around code quality? Are there good examples of code reviews? Are there enforced coding standards?_
```

Comment:
Yes? No? Missing?

Reply:
Will add an answer later today or this evening.